en/middleware/xrce_dds.md

```diff
@@ -233,6 +233,9 @@ At time of writing there is no PX4 parameter for setting the address of the remo
 Therefore for Ethernet connections you will _have_ to use a command to start the client.
 :::
 
+On the simulator the client is automatically started on localhost UDP port 8888.
+Moreover, the startup scripts take into account [multi vehicle simulations](../ros/ros2_multi_vehicle.md) and their configuration.
```

Comment:
FYI these bits might lead to some conflict with #2352

Reply:
@beniaminopozzan The second line confuses me a bit. We start using parameters most of the time "in future" so is this saying that parameters won't take into account multi vehicle, or that if I write a start up script it has to take into account multi-vehicle simulation?

Reply:
@hamishwillee Yes, most likely they will

At the moment you can do almost everything with parameters: set the IP, port, session key. However, namespaces cannot be set with parameters and users need to start the agent manually and use either the `-s` argument or `PX4_MICRODDS_NS`. There is no other way.
To have multi vehicle setups (simulation or real vehicles) you need to set namespaces.
Therefore:

- Parameters won't take into account multi vehicle? Yes, they won't. And this behavior will not change until string parameter will be supported in PX4.
- If I write a start up script it has to take into account multi-vehicle simulation? If you write a startup script and you want to perform multi-vehicle simulation, then yes, your script must handle them. If you don't want to perform multi-vehicle simulation then no, your script doesn't have to handle them.

Reply:
Thanks for the clarification. I guess it is a bit confusing because we're talking about "the startup scripts" here as though users know what specific scripts you mean.

How about this?

```suggestion
On the simulator the client is automatically started on localhost UDP port 8888.

Note that startup commands are used in startup scripts for [multi vehicle simulations](../ros/ros2_multi_vehicle.md), enabling the setting of appropriate instance ids and DDS namespace.
```

Reply:
As

> On the simulator the client is automatically started on localhost UDP port 8888.

is already mentioned in the first note for "Starting the Client", another possibility is

```suggestion
The simulator [startup logic](../concept/system_startup.md) ([init.d-posix/rcS](https://github.com/PX4/PX4-Autopilot/blob/main/ROMFS/px4fmu_common/init.d-posix/rcS)) uses the client startup commands for single and [multi vehicle simulations](../ros/ros2_multi_vehicle.md), enabling the setting of appropriate instance ids and DDS namespaces.
```

Reply:
Yes.